src/clj/commiteth/scheduler.clj

```diff
@@ -342,7 +342,8 @@
   (do
     (log/debug "run-1-min-interval-tasks" time)
     (run-tasks
-     [update-open-issue-usd-values])
+     [update-open-issue-usd-values
+      eth/check-bot-account-eth-threshold])
```

Comment:
This should run less frequently as the scheduler thread very busy already. Hourly is enough.